core/src/main/java/org/radargun/stages/test/legacy/LegacyStressor.java

```diff
@@ -94,8 +96,13 @@ private void runInternal() {
             Operation operation = operationSelector.next(random);
             try {
                logic.run(operation);
+               if (delayBetweenRequests > 0)
```

Comment:
I know the implementation does b); my concern is that you can't quantify the difference int latency between two impls/versions/configs when these two have different throughput.
In my view think time makes sense when client executes a sequence of operations, but in this test the requests don't present any sequence, therefore, I would expect that you are interested in latency when executing 1000 req/s, 2000 req/s etc. That's why we've used fixed schedule in the past; the throughput was close to number of threads * duration/period. It will suffer from coordinated omission, but you could at least compensate by setting the `requestStartTime` to the scheduled time (that wasn't done in the past since I wanted to fix everything right away, not realizing that it's impossible with sync API).